src/commands/command-fns/thanks.js

```diff
@@ -9,8 +10,44 @@ const {
   getMessageLink,
 } = require('../utils')
 
-async function thanks(message) {
-  const args = getCommandArgs(message.content)
+async function getThanksHistory() {
+  const response = await got.get(
+    `https://api.github.com/gists/${process.env.GIST_REPO_THANKS}`,
+    {
+      headers: {
+        Authorization: `token ${process.env.GITHUB_TOKEN}`,
```

Comment:
I only enabled the "gist" permission for this token, so I named it accordingly.

```suggestion
        Authorization: `token ${process.env.GIST_BOT_TOKEN}`,
```